### PR TITLE
feat(skyrim-platform): add texts API

### LINKED ISSUE #431: Add texts SP API
Add to SP API for rendering texts: let id = createText, destroyText (id), get / setTextPos, get / setTextString, get / setTextColor, destroyAllTexts 

Depends on #430

### PR DESCRIPTION
**Pull request by Retr0Wawe**
closes #431

### CODE REVIEW

**Review by Pospelove — COMMENTED** [review]
1. Plz no commented code and no //Getters comment
2. We need to test if Cyrillic symbols work
3. We need some kind of texts limit. 1000 or 10000. some reasonable number

TODO: for me - think about threading

File: skyrim-platform/src/platform_se/skyrim_platform/main.cpp [hunk 1]
```diff
@@ -63,8 +64,15 @@ extern CallNativeApi::NativeCallRequirements g_nativeCallRequirements;
 
 void GetTextsToDraw(TextToDrawCallback callback)
 {
-  callback(TextToDraw());
-  callback(TextToDraw());
+  auto text = &TextsCollection::GetSinglton();
+
+  try {
```
Comment (Pospelove) [inline skyrim-platform/src/platform_se/skyrim_platform/main.cpp:69]: this try/catch doesn't seem useful. consider removing it. or explain what kind of exceptions does it catch

**Review by Pospelove — CHANGES_REQUESTED** [review]
Please also do the following:
1. Fix CI errors
2. Add file with feature desctiption here (https://github.com/skyrim-multiplayer/skymp/tree/main/docs/release/dev)
(your description goes to changelog)
3. Add `texts.md` here https://github.com/skyrim-multiplayer/skymp/tree/main/docs/skyrim_platform (see win32, browser for reference)

File: skyrim-platform/src/platform_se/skyrim_platform/TextsCollection.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  void SetTextPos(int textId, float xPos, float yPos);
+
+  void SetTextString(int textId, std::string str);
+
+  void SetTextColor(int textId, std::array<double, 4> color);
+
+  void DestroyAllTexts();
+
+  static TextsCollection& GetSinglton() noexcept;
```
Comment (nic11) [inline skyrim-platform/src/platform_se/skyrim_platform/TextsCollection.h:27]: typo:
```suggestion
  static TextsCollection& GetSingleton() noexcept;
```

Comment (Retr0Wawe) [reply skyrim-platform/src/platform_se/skyrim_platform/TextsCollection.h:27]: thanks

**Review by Pospelove — CHANGES_REQUESTED** [review]
1) Add file with feature desctiption here (https://github.com/skyrim-multiplayer/skymp/tree/main/docs/release/dev)
(your description goes to changelog)
2) Add texts.md here https://github.com/skyrim-multiplayer/skymp/tree/main/docs/skyrim_platform (see win32, browser for reference)
3) Write TypeScript definitions for the new API here https://github.com/skyrim-multiplayer/skymp/blob/main/skyrim-platform/src/platform_se/codegen/convert-files/Definitions.txt

**Review by Pospelove — APPROVED** [review]

**Pull request merged**